### PR TITLE
Add support for chi router

### PR DESCRIPTION
**Pull request by dcu**

### CODE REVIEW

File: frameworks/Go/chi/chi.dockerfile [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM techempower/go-lang:0.1
```
Comment (nbrady-techempower) [inline frameworks/Go/chi/chi.dockerfile]: FYI, we're going to be using official repos for languages now. Would you mind changing this to:

`FROM golang:1.10.1`

or whichever suits your needs. Thanks! the rest looks good.

**Comment by dcu** [comment]
@nbrady-techempower this is ready!

**Comment by nbrady-techempower** [comment]
@dcu looks like it's running, your fortune test failed https://travis-ci.org/TechEmpower/FrameworkBenchmarks/jobs/364677453

**Comment by dcu** [comment]
yeah thanks

**Comment by dcu** [comment]
@nbrady-techempower any idea why this didn't appear in the latest run here: https://tfb-status.techempower.com/ ? is something missing?

**Comment by nbrady-techempower** [comment]
that run took 68 hours to complete, it may not have been in master before the run was prepping. look for it in the next full rune

**Pull request merged**